app/Jobs/MigrationWikiCreate.php

```diff
@@ -105,6 +106,12 @@ private function createWiki($user, $wikiDetails, $wikiDb) {
             $wikiDomain->wiki_id = $wiki->id;
             $wikiDomain->save();
 
+            // Create k8s ingress if the wiki uses a custom domain
+            if ( ! WikiController::isSubDomain($wikiDetails->domain)) {
+                // TODO: should this be done synchronously with dispatch_sync(), or asynchronously with dispatch()?
```

Comment:
Couldn't we do this outside of transaction to only happen if the whole thing is successful? I don't think in we'd need to do it in sync with this job then either? looks like you could use `$wiki->domain` if set maybe?